src/Instractor.jl

```diff
@@ -15,7 +15,7 @@ end
 function read_entry(file_ptr::GZip.GZipStream)
     he = String(chomp(readline(file_ptr)))
```

Comment:
Is `chomp` needed here? Hasn't `readline` already removed the trailing newline character?